subset/security/security_passwords/src/main/resources/defaultPasswords.json

```diff
@@ -1,78 +1,93 @@
-    "Passwords":"1234,admin,apc,cisco,connect,default,letmein,manager,none,pass,password,ppmax2011,root,sanfran,security,sitecom,sys,system,tomcat,turnkey,user,wampp,xampp"
+    "Usernames":["admin","apc","axis2","cisco","connect","manager","newuser","pass","private","root","security","sitecom","sys","system","tomcat","user","wampp","xampp","xampp-dav-unsecure"],
+    "Passwords":["1234","admin","apc","cisco","connect","default","letmein","manager","none","pass","password","ppmax2011","root","sanfran","security","sitecom","sys","system","tomcat","turnkey","user","wampp","xampp"
+      ]
   },
   "MSSQL":{
-    "Usernames":"Administrator,ARAdmin,entldbdbo,entldbreader,mon_user,probe,repl_publisher,repl_subscriber,sa,WinCCConnect",
-    "Passwords":"2WSXcder,AR#Admin#,blank,dbopswd,pass,pass1,password,rdrpswd"
+    "Usernames":["Administrator","ARAdmin","entldbdbo","entldbreader","mon_user","probe","repl_publisher","repl_subscriber","sa","WinCCConnect"],
+    "Passwords":["2WSXcder","AR#Admin#","blank","dbopswd","pass","pass1","password","rdrpswd"
+    ]
   },
   "MySQL":{
-    "Usernames":"admin,mysql,root",
-    "Passwords":"blank,pass,pass1,password,vicidia1now"
+    "Usernames":["admin","mysql","root"],
+    "Passwords":["blank","pass","pass1","password","vicidia1now"
+    ]
   },
   "PostgreSQL":{
-    "Usernames":"admin,postgres,scott,tom",
-    "Passwords":"admin,password,postgres,tiger"
+    "Usernames":["admin","postgres","scott","tom"],
+    "Passwords":["admin","password","postgres","tiger"
+    ]
   },
   "SMB":{
-    "Usernames":"backuphelpdesk",
-    "Passwords":"backuphpinvent"
+    "Usernames":["backuphelpdesk"],
+    "Passwords":["backuphpinvent"
+    ]
   },
   "SNMP":{
-    "Usernames":"<BLANK>",
-    "Passwords":"0392a0,1234,2read,4changes,access.adm,Admin,admin,agent,agent_steal,all,allprivate,allpublic,ANYCOM,apc,bintec,blue,c,C0de,cable-d,canon_admin,cc,CISCO,cisco,community,core,CR52401,debug,default,dilbert,enable,field,field-service,freekevin,fubar,guest,hello,hp_admin,IBM,ibm,ILMI,ilmi,Intermec,intermec,internal,l2,l3,manager,mngt,monitor,netman,network,NoGaH$@!,none,openview,OrigEquipMfr,pass,password,pr1v4t3,private,PRIVATE,Private,proxy,publ1c,public,PUBLIC,Public,read,read-only,read-write,readwrite,red,regional,rmon,rmon_admin,ro,root,router,rw,rwa,s!a@m#n$p%c,san-fran,sanfran,scotty,SECRET,Secret,secret,SECURITY,Security,security,seri,SNMP,snmp,SNMP_trap,snmpd,snmptrap,solaris,SUN,sun,superuser,SWITCH,Switch,switch,SYSTEM,System,system,tech,TENmanUFactOryPOWER,TEST,test,test2,tiv0li,tivoli,trap,world,write,xyzzy,yellow"
+    "Usernames":["<BLANK>"],
+    "Passwords":["0392a0","1234","2read","4changes","access.adm","Admin","admin","agent","agent_steal","all","allprivate","allpublic","ANYCOM","apc","bintec","blue","c","C0de","cable-d","canon_admin","cc","CISCO","cisco"

```

Comment:
nit: now you can use linebreaks instead of one long line :-) -- up to you if you want to change it or not.

Reply:
Added line breaks to the json file, just so it doesn't run off the screen